todolist/main.js

```diff
@@ -90,35 +92,46 @@ sendTextBox.addEventListener("keydown", (e) => {
   }
 });
 
+window.addEventListener("click", (e) => {
+  if (e.target === modalBox) {
+    disenableModalDiv();
+  }
+});
+
+modalBoxRmBtn.addEventListener("click", () => {
```

Comment:
```suggestion
modalBoxRmBtn.addEventListener('click', () => {
```

그리고 이거 prettier 때문인 것 같은데 js에서 문자열은 '' 사용이 컨벤션입니다.